api/src/main/scala/hmda/api/model/EditResults.scala

```diff
@@ -0,0 +1,11 @@
+package hmda.api.model
+
+case class LarId(loanId: String)
+case class LarEditResult(lar: LarId)
+case class EditResult(edit: String, lars: Seq[LarEditResult])
+case class EditResults(edits: Seq[EditResult])
+case object EditResults {
+  def empty: EditResults = EditResults(Nil)
```

Comment:
I'm using this in my single-type edits endpoint, so I'm fine with leaving this line untested for now.